changelogs/fragments/54-remove_tests_sanity_requirements.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
No need for a changelog. The file we are removing wasn't ever used by anything

Reply:
@gundalow if we don't add the fragment, CI will fail because it requires a fragment (`trivial` in this case)

Reply:
Oh, I didn't know that, thanks for the info